config/config.go

```diff
@@ -95,6 +98,7 @@ type Config struct {
 	TempStoragePath             string `toml:"tmp-storage-path" json:"tmp-storage-path"`
 	OOMAction                   string `toml:"oom-action" json:"oom-action"`
 	MemQuotaQuery               int64  `toml:"mem-quota-query" json:"mem-quota-query"`
+	MemQuotaStatistic           int64  `toml:"mem-quota-statistic" json:"mem-quota-statistic"`
```

Comment:
where to valid `MemQuotaStatistic `? what if input a negtive value?

Reply:
MemQuotaStatistic must be greater than `DefMinQuotaStatistic`